src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((item) => {
+    const child = item.querySelector('ul');
+
+    if (!child) {
+      return;
+    }
+
+    const span = document.createElement('span');
+
+    span.textContent = item.firstChild.textContent.trim();
+    item.firstChild.replaceWith(span);
+
+    child.style.display = 'block';
```

Comment:
You are setting the display property of child elements to 'block' by default. This could cause all child elements to be visible at the start, which might not be the desired behavior. Consider setting it to 'none' initially if you want the child elements to be hidden by default.